media/css/careers/modules/bottom.scss

```diff
@@ -9,49 +9,57 @@ $image-path: '/media/protocol/img';
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
 
 .c-careers-bottom {
-    .mzp-c-split {
-        overflow: hidden;
-        padding-top: 0;
+    margin-top: $layout-md;
 
-        .mzp-c-split-container {
-            margin-bottom: -180px;
-        }
+    .mzp-c-split.mzp-t-split-nospace {
+        padding-bottom: 0;
+    }
+
+    .c-careers-socials {
+        display: flex;
+        justify-content: center;
+        margin: $spacing-lg auto;
+
+        li {
+            margin: 0 $spacing-sm;
+
+            a {
+                @include image-replaced;
+                background-position: center center;
+                background-repeat: no-repeat;
+                background-size: 20px 20px;
+                border-bottom: 2px solid transparent;
+                border-radius: $border-radius-sm;
+                display: inline-block;
+                height: 22px;
+                padding: $spacing-xs;
+                text-decoration: underline;
+                width: 22px;
+
+                &:hover,
+                &:focus,
+                &:active {
+                    background-color: cp.$blue-light;
+                }
 
-        .c-careers-socials {
-            display: flex;
-            margin-top: $spacing-lg;
-
-            li {
-                margin-right: $spacing-md;
-
-                a {
-                    display: inline-block;
-                    width: 22px;
-                    height: 20px;
-                    border-bottom: 2px solid transparent;
-                    text-decoration: underline;
-                    position: relative;
-
-                    &.c-careers-instagram {
-                        background-image: url('#{$image-path}/icons/social/instagram/black.svg');
-                        background-size: contain;
-                        background-repeat: no-repeat;
-                        background-origin: content-box;
-                    }
-
-                    &.c-careers-twitter {
-                        background-image: url('#{$image-path}/icons/social/twitter/black.svg');
-                        background-size: contain;
-                        background-repeat: no-repeat;
-                    }
-
-                    &.c-careers-glassdoor {
-                        background-image: url('/media/img/careers/icons/social/glassdoor/black.svg');
-                        background-size: contain;
-                        background-repeat: no-repeat;
-                    }
+                &.c-careers-instagram {
+                    background-image: url('#{$image-path}/icons/social/instagram/black.svg');
+                    background-size: contain;
+                    background-repeat: no-repeat;
```

Comment:
**nit:** We can remove these two lines because the `background-size: contain` is enlarging the icon, whereas the `background-repeat` is being placed above in the `a` tag 
```suggestion

```